site/cds_rdm/inspire_harvester/writer.py

```diff
@@ -16,47 +16,82 @@
 from invenio_search.engine import dsl
 from invenio_vocabularies.datastreams.errors import WriterError
 from invenio_vocabularies.datastreams.writers import BaseWriter
+from marshmallow import ValidationError
 
 
 class InspireWriter(BaseWriter):
     """INSPIRE writer."""
 
     def _write_entry(self, entry, *args, **kwargs):
+        """Write entry to CDS."""
         inspire_id = entry["id"]
+        # errors = []
         existing_records = self._get_existing_records(inspire_id)
         multiple_records_found = existing_records.total > 1
         should_update = existing_records.total == 1
         should_create = existing_records.total == 0
 
         existing_records_hits = existing_records.to_dict()["hits"]["hits"]
         existing_records_ids = [hit["id"] for hit in existing_records_hits]
-
+        current_app.logger.info(
+            f"IDs of existing records found: {existing_records_ids}."
+        )
         if multiple_records_found:
-            raise WriterError(
-                f"More than 1 record found with INSPIRE id {inspire_id}. CDS records found: {', '.join(existing_records_ids)}"
+            # errors.append(f"{existing_records.total} records found on CDS with the same INSPIRE ID ({inspire_id}). Found records ids: {', '.join(existing_records_ids)}.")
```

Comment:
could we remove these comments? maybe you can keep it in a separate branch if you don't want to lose it?